planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -132,6 +132,9 @@ void NormalLaneChange::update_transient_data()
   transient_data.is_ego_near_current_terminal_start =
     transient_data.dist_to_terminal_start < transient_data.max_prepare_length;
 
+  updateStopTime();
+  transient_data.is_ego_stuck = isVehicleStuck();
```

Comment:
i`sVehicleStuck()` needs to access `filtered_objects`, and currently `update_filtered_objects()` is called after `update_transient_data()`.

Reply:
replace the arrangement here
https://github.com/autowarefoundation/autoware.universe/pull/8999/commits/91b60e96a801e0bb3b451d91687b6e47519f5505